environments/dev/outputs.tf

```diff
@@ -0,0 +1,21 @@
+output "lambda_arn" {
```

Comment:
Overall/beauty comment; 
add empty lines between resources/outputs blocks, it improves visibility